add clarification in ADR overview page (#161)
wmde/wikit: docs/src/doc/decisions-and-ADRs/01_overview.stories.mdx

# Decisions and ADRs

> Crucial decisions are documented in the form of Architecture Decision Records (or ADRs). Althought ADRs usually cover topics exclusive to the dev realm, the clarity in which they are written and structured makes them accessible to all the members of our cross-functional team.

Author: ```suggestion
Crucial decisions are documented in the form of Architecture Decision Records (or ADRs). Althought ADRs usually cover topics exclusive to the dev realm, the clarity in which they are written and structured makes their content accessible to all the members of our cross-functional team.
```

> Crucial decisions are documented in the form of Architecture Decision Records (or ADRs). Althought ADRs usually cover topics exclusive to the dev realm, the clarity in which they are written and structured makes their content accessible to all the members of our cross-functional team.

Reviewer: ```suggestion
Crucial decisions are documented in the form of Architecture Decision Records (ADRs). Although ADRs usually cover topics exclusive to the dev realm, the clarity in which they are written and structured makes their content accessible to all the members of our cross-functional team. You can find all development and design related decisions in this folder. 
```

> Crucial decisions are documented in the form of Architecture Decision Records (or ADRs). Althought they usually cover topics exclusive to the dev realm, ADRs are very clearly written and structured in a standardized that generally makes them accessible to all the members of our cross-functional team.

Author: ```suggestion
Crucial decisions are documented in the form of Architecture Decision Records (or ADRs). Althought ADRs usually cover topics exclusive to the dev realm, the clarity in which they are written and structured makes them accessible to all the members of our cross-functional team.
```
